src/modules/src/outlierFilter.c

```diff
@@ -102,6 +101,84 @@ bool outlierFilterValidateTdoaSteps(const tdoaMeasurement_t* tdoa, const float e
   return sampleIsGood;
 }
 
+// Simple TDoA outlier filter
+bool outlierFilterValidateTdoaSimple(const tdoaMeasurement_t* tdoa) {
+  return isDistanceDiffSmallerThanDistanceBetweenAnchors(tdoa);
+}
+
+
+// Integrator TDoA outlier filter -------------------------------------
+// This filter uses an "integrator" to track the ratio of good/samples. Samples with an errors (measurement - predicted)
+// larger than the acceptance level will be discarded. The acceptance level is based on the integrator, with a
+// hysteresis to avoid rapid changes. When the filter is open, all samples are let through, usually at startup to let
+// the kalman filter converge. When the filter is closed, only samples with an error < the acceptance level, this should
+// be most of the time.
+// The acceptance level is based on the standard deviation used for the tdoa samples, which should be based on the
+// noise level in the system.
+
+
+// The maximum size of the integrator. This size determines the time [in ms] needed to open/close the filter.
+static const float INTEGRATOR_SIZE = 300.0f;
```

Comment:
I'm wondering if we should make a struct for these values and initialize the avalues when the kalman filter is initialize? What do you think? It will make it more suitable for wrapping and the bindings?

Reply:
Also it would make it possible to use this filter for something else